get-additional-demos.sh

```diff
@@ -14,16 +14,22 @@ user=${V3IO_USERNAME}
 
 USAGE="\
 $SCRIPT:
-Retrieves latest-release demos from the mlrun/demos GitHub repo.
+Retrieves updated demos from the mlrun/demos GitHub repository.
 USAGE: ${SCRIPT} [OPTIONS]
 OPTIONS:
   -h|--help   -  Display this message and exit.
-  -b|--branch -  Branch name (default is branch equivalent to the current MLRun version).
-  -u|--user   -  Username, which determines the directory to which to copy the files - /v3io/users/<username>.
-                 Default = \$V3IO_USERNAME (if set to a non-empty string).
-  --mlrun-ver -  Determine the branch based on the MLRun version, unless -b/--branch option is specified.
-  --dry-run   -  Do not update the files, but rather show the pending changes.
-  --no-backup -  Do not backup the current demos directory."
+  -b|--branch -  Git branch name. Default: The latest release branch that
+                 matches the version of the running MLRun service.
```

Comment:
Default is the version that matches the mlrun python package installed in the current python environment.

Reply:
I edited the help text for the `-b|--branch` and `--mlrun-ver` flags and the get-demos doc in **welcome.ipynb**/**README.md** — see commit 6f3279b9.